src/main/java/seedu/address/model/person/Person.java

```diff
@@ -128,7 +136,7 @@ public boolean equals(Object other) {
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, birthday, instrument, tags, attendances);
+        return Objects.hash(name, phone, email, address, birthday, matriculationYear, tags, attendances);
```

Comment:
Was `instrument` deleted accidentally here?

Reply:
Thanks for the suggestion! I have updated the code accordingly with the suggested changes.